src/ucp/core/ucp_worker.c

```diff
@@ -32,6 +32,8 @@
 #include <sys/epoll.h>
 
 
+#define UCP_WORKER_KEEPALIVE_ITER_SKIP 32
```

Comment:
@yosefe make this parameter run-time configurable?

Reply:
IMO, the benefit of making it fast check (mod by constant) outweighs the value of making it configurable: we already have a configurable timeout, this is just to reduce the cost of timer check